packages/ui/primitives/command.tsx

```diff
@@ -32,7 +32,11 @@ type CommandDialogProps = DialogProps & {
 const CommandDialog = ({ children, commandProps, ...props }: CommandDialogProps) => {
   return (
     <Dialog {...props}>
-      <DialogContent className="overflow-hidden p-0 shadow-2xl">
+      <DialogContent
+        className="w-11/12 items-center overflow-hidden rounded-lg p-0 shadow-2xl lg:mt-0"
```

Comment:
what's the reason for the changes from `command` and `dialog` files? Trying to understand what has changed. :-D

Reply:
The overlay had a feature to have its own classnames, so instead of writing it in the main component, I thought why dont we just extend it and it might be of use for other uses of DialogContent in the future

Reply:
Hmmm, got it!